src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -73,6 +76,7 @@ const ProfileHeader = ({
     'font-family--sans',
     'font-size--base',
     'font-weight--normal',
+    'line-height--3',
```

Comment:
This utility class sets the `line-height` to `1.3` (instead of the default `1.5`).